assets/css/mob-home.css

```diff
@@ -0,0 +1,190 @@
+@media screen and (max-width: 768px) {
+  @import url("https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap");
+
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  .header-below {
+    padding: 0;
+  }
+
+  .header-below-list {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: flex-start;
+    gap: 2rem;
+  }
+
+  .header-below-content {
+    font-size: 30px;
+    font-weight: 400;
+    color: #fff;
+  }
+
+  .cc-btn {
+    border: none;
+    padding: 0;
+  }
+
+  body {
+    font-family: 'Lato', sans-serif;
+  }
+
+  .mob-header {
+    display: flex;
+    align-items: center;
+    width: 100%;
+    margin-left: 23px;
+  }
+
+  .links {
+    display: flex;
+    justify-content: center;
+    padding: 0;
+  }
+
+  .links-hamburger {
+    width: 100%;
+  }
+
+  .conference-content {
+    padding: 109px 20px 79px;
+  }
+
+  .conference-intro {
+    font-size: 40px;
+    font-weight: 500;
+    line-height: 38px;
+  }
+
+  .first-text {
+    font-size: 15px;
+    font-weight: 300;
+    text-align: left;
+  }
+
+  .program-details {
+    display: flex;
+    flex-direction: column;
+    padding: 0;
+  }
+
+  .program-content {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    gap: 22px;
+  }
+
+  .program-title {
+    width: 15%;
+    line-height: 15px;
+  }
+
+  .program-text {
+    text-align: left;
+    width: 50%;
+  }
```

Comment:
- [ ] Please ensure that your program details texts do not overlap each other, please follow the design guidelines, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustrations of yours and the given Design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Student mobile main program](https://github.com/Kwamay/Capstone-Project/assets/43528594/19c12f77-eda1-4aad-9f47-61c01a9a237a)


</td>
      <td>

![Design mobile main program](https://github.com/Kwamay/Capstone-Project/assets/43528594/49fcbf7f-606d-458e-bcb6-9e087667b93f)

</td>
    </tr>
  </table>